content/guides/integration-guides/csi-integration/soda-csi-pnp/_index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: SODA Integration with Vendor CSI Drivers via SODA CSI Plug-N-Play
+menuTitle: "SODA CSI Plug-N-Play"
+description: "A guide for integrating SODA with Vendor CSI Driver via unified Plugin."
+weight: 10
+disableToc: false
+tags: ["integration guide", "kubernetes csi", "plug-n-play", "unified plugin"] 
+---
+
+SODA CSI Plug-N-Play is a mechanism by which users can use the heterogeneous storage vendors in a unified way, 
+Users need to define the requirements while creating SODA Profile, 
```

Comment:
Info is displayed as comma seperated points, pls check
![image](https://user-images.githubusercontent.com/30930862/110905939-476e9e00-8331-11eb-8012-484626bf86e4.png)

Reply:
this gets rendered properly in website, website takes up a new line only when it gets double space https://docs.sodafoundation.io/guides/integration-guides/csi-integration/